packages/dx-react-scheduler/docs/reference/current-time-indicator.md

```diff
@@ -0,0 +1,54 @@
+# CurrentTimeIndicator Plugin Reference
+
+A plugin that renders the current time indicator and shades that cover appointments and timetable cells up to the current time.
+
+## Import
+
+Use the following statement to import the plugin with embedded theme components:
+
+```js
+import { CurrentTimeIndicator } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+You can import the themeless plugin if you want to use custom components:
+
+```js
+import { CurrentTimeIndicator } from '@devexpress/dx-react-scheduler';
+```
+
+## User Reference
+
+### Dependencies
+
+- [DayView](day-view.md) [Optional]
+- [WeekView](week-view.md) [Optional]
+- [MonthView](month-view.md) [Optional]
+- [Appointments](appointments.md) [Optional]
+- [DragDropProvider](drag-drop-provider.md) [Optional]
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+updateInterval | number | 60000 | An interval in milliseconds that specifies how frequently the indicator's position is updated.
+shadePastAppointments | boolean | `false` | Specifies whether past appointments should be shaded.
+shadePastCells | boolean | `false` | Specifies whether past cells should be shaded.
```

Comment:
past appointments? past cells? Do you mean 'previous'?

Reply:
We are describing a current time indicator, so past appointments are all appointments that ended before current time; similarly, past cells are timetable cells that represent time intervals that ended before current time.

Reply:
Then you should use 'previous', since 'past' could indicate any previous time interval.

Reply:
```suggestion
shadePastCells | boolean | `false` | Specifies whether previous cells should be shaded.
```